src/modules/AieraChat/components/Messages/index.tsx

```diff
@@ -204,120 +206,124 @@ export function Messages({
         }
     }, []);
 
-    const onSubmit = useCallback(
+    const handleSubmit = useCallback(
         (prompt: string) => {
-            const myMessage: ChatMessagePrompt = {
-                id: `${idCounter++}`,
-                ordinalId: `${idCounter++}`,
-                prompt,
-                type: ChatMessageType.PROMPT,
-                status: ChatMessageStatus.COMPLETED,
-                timestamp: '',
-            };
-            virtuosoRef.current?.data.append([myMessage], ({ scrollInProgress, atBottom }) => {
-                return {
-                    index: 'LAST',
-                    align: 'end',
-                    behavior: atBottom || scrollInProgress ? 'smooth' : 'auto',
-                };
-            });
-
-            if (sources.length === 0) {
-                const newKey = `${idCounter++}`;
-                const sourceMessage: ChatMessageSources = {
-                    id: newKey,
-                    ordinalId: newKey,
-                    confirmed: false,
-                    sources: [],
-                    timestamp: '',
-                    prompt,
-                    status: ChatMessageStatus.PENDING,
-                    type: ChatMessageType.SOURCES,
-                };
-                virtuosoRef.current?.data.append([sourceMessage], ({ scrollInProgress, atBottom }) => {
-                    return {
-                        index: 'LAST',
-                        align: 'end',
-                        behavior: atBottom || scrollInProgress ? 'smooth' : 'auto',
+            onSubmit(prompt)
```

Comment:
This just does the message stuff after the chat/prompt is created. The messages will be replaced with ably.